packages/orbit-components/src/CountryFlag/index.tsx

```diff
@@ -6,19 +6,18 @@ import cx from "clsx";
 import { baseURL, CODES, SIZE_WIDTHS, SIZES } from "./consts";
 import type { Props } from "./types";
 
-function getCountryProps(code?: string, name?: string) {
+function getCountryProps(code?: string) {
   const codeNormalized = code ? code.toUpperCase().replace("-", "_") : "UNDEFINED";
   const countryCodeExists = codeNormalized in CODES;
 
   if (!countryCodeExists) console.warn(`Country code not supported: ${code}`);
 
   const countryCode = countryCodeExists ? CODES[codeNormalized] : CODES.UNDEFINED;
-  const countryName = countryCode === CODES.UNDEFINED && !name ? "Undefined" : name;
-  return { code: countryCode, name: countryName };
+  return { code: countryCode };
 }
 
-const CountryFlag = ({ dataTest, size = SIZES.MEDIUM, id, role = "img", ...props }: Props) => {
-  const { code, name } = getCountryProps(props.code, props.name);
+const CountryFlag = ({ dataTest, size = SIZES.MEDIUM, id, role, name, ...props }: Props) => {
```

Comment:
why did you decide to set the value of `alt` in a `name` prop? it looks that `name` is used only on line 40 as a value of `alt` attribute. why didn't you keep it as a prop called `alt`?

Reply:
Because I did not want to introduce a breaking change

Reply:
why not? we're going to have BC anyways. :D 

ie:
- https://github.com/kiwicom/orbit/pull/4686/commits/18d5e4fef119fed48cd967dcc22f4077f01bf6b6
- https://github.com/kiwicom/orbit/pull/4691/commits/9d1611e6bbd522887cec91df8304d35c1f6e990f

Reply:
Because we already have a prop called "name" that was being applied to `alt` property… There's no point in complicating